modules/core/src/main/java/org/apache/synapse/mediators/builtin/PropertyMediator.java

```diff
@@ -145,6 +150,15 @@ public boolean mediate(MessageContext synCtx) {
 				
                 synCtx.setProperty(name, resultValue);
 
+            } else if (XMLConfigConstants.SCOPE_TRACE.equals(scope)) {
+                //Setting property value into the propertyValue variable for tracing purposes
+                if (resultValue != null ) {
+                    if (resultValue instanceof OMElement) {
+                        ((OMElement) resultValue).build();
+                    }
+                    //Converted to string since only used to display as a span tag
+                    propertyValue = resultValue.toString();
```

Comment:
Following this approach can we add multiple property values to a single span?

Reply:
This is to support adding the value of each property mediator which is in trace scope to span tags previously only property mediator name was added.